.travis-ci.sh

```diff
@@ -72,11 +72,11 @@ EOF
   export PATH=$PREFIX/bin:$PATH
   $MAKE world.opt
   $MAKE ocamlnat
-  cd testsuite
+  cd testsuite || exit 1
```

Comment:
This change is unnecessary - the entire script is running via `bash -e` and this directory will exist anyway - no one is going to delete the entire testsuite and that alteration go otherwise unnoticed!

Reply:
In `.travis.yml` it's better to use:

```yml
script: ./travis.sh
```

And so, to add `set -ex` in the `.sh` script. Will update this.

Reply:
This requires the scripts to be invoked using the execute bit, which is intentionally not the case at the moment.

Reply:
Oh no, having reviewed the changes again, I see you only did this for the Travis script, not AppVeyor.